### PR TITLE
Returns old adrenal implants

### PR DESCRIPTION
**Pull request by SuperSlayer0**
## About The Pull Request

Reverts cancer TG adrenals nerfs and removal

## Why It's Good For The Game

Now you have an alternative to stimpacks as a traitor

## Changelog
:cl: SuperSlayer
add: Added adrenal implants
/:cl:

### CODE REVIEW

File: code/game/objects/items/implants/implant_misc.dm [hunk 1]
```diff
@@ -109,3 +109,43 @@
...
+				<b>Integrity:</b> Implant can only be used three times before reserves are depleted."}
+	return dat
+
+/obj/item/implant/adrenalin/activate()
+	. = ..()
+	uses--
+	to_chat(imp_in, "<span class='notice'>You feel a sudden surge of energy!</span>")
+	imp_in.SetStun(0)
+	imp_in.SetKnockdown(0)
+	imp_in.SetUnconscious(0)
```
Comment (kriskog) [inline code/game/objects/items/implants/implant_misc.dm:135]: Knockdown is here twice, unconscious is a bit iffy. I doubt you can use this while unconscious, and if you can you probably shouldn't be able to.

File: code/modules/movespeed/modifiers/reagent.dm [hunk 2]
```diff
@@ -10,6 +10,9 @@
 /datum/movespeed_modifier/reagent/pepperspray
 	multiplicative_slowdown = 0.25
 
+/datum/movespeed_modifier/reagent/badstims
+	multiplicative_slowdown = -0.45
+
```
Comment (kriskog) [inline code/modules/movespeed/modifiers/reagent.dm:15]: If its the old one, this one does not exist.

File: code/game/objects/items/implants/implant_misc.dm [hunk 3]
```diff
@@ -109,3 +109,43 @@
...
+	imp_in.SetParalyzed(0)
+	imp_in.SetImmobilized(0)
+	imp_in.adjustStaminaLoss(-75)
+	imp_in.SetKnockdown(0)
+	imp_in.set_resting(FALSE)
+	imp_in.update_mobility()
+
+	imp_in.reagents.add_reagent(/datum/reagent/medicine/synaptizine, 10)
+	imp_in.reagents.add_reagent(/datum/reagent/medicine/omnizine, 10)
+	imp_in.reagents.add_reagent(/datum/reagent/medicine/stimulants, 10)
```
Comment (kriskog) [inline code/game/objects/items/implants/implant_misc.dm:145]: These should be lowered. This is 200 seconds of omnizine healing, 100 seconds of speedy gonzales. Omnizine no higher than 1.5u, stimulants no higher than 2u.

Comment (SuperSlayer0) [reply code/game/objects/items/implants/implant_misc.dm]: Actually no, this is stupid. Who will buy a 3 use implant with 2u of stimulants and 1.5u of omnizine for 8 TC if you can just buy a stimpack that costs lower(5 TC) and contains 50u of stimulants. So having 10 of each reagents is okay.

Comment (kriskog) [reply code/game/objects/items/implants/implant_misc.dm]: You pay for the ability to use it after getting stunned, as well as the instant effects that the stimpack does not have.
In fact, the omnizine doesn't fit in this at all, and should be removed. Its a remnant from the times before cobbychems.

Comment (Pugie) [reply code/game/objects/items/implants/implant_misc.dm]: I think it's to deal with the synaptizine toxin.

**Comment by SuperSlayer0** [comment]
Fixed bugs and lowered amount of chems injected to 5 of each

**Comment by EdgeLordExe** [comment]
Doesn't fit our server

**Pull request closed without merging**